car-statistics/app/services/utils.py

```diff
@@ -96,7 +95,13 @@ def attributes(file_path):
     :param file_path: file which will be added to DB
     :return: json with file attributes
     """
-    attrbts = {'loaded': 'new'}
+    attrbts = dict()
+    attrbts['name'] = 'name'
+    file_size_MB = os.path.getsize(file_path)/(1024*1024.0) #in MBytes
+    attrbts['size'] = math.ceil(file_size_MB*100)/100       # round 2 decimals after poin
+    attrbts['rows'] = 0
+    attrbts['date'] = time.time()
```

Comment:
You should use same approach https://github.com/lv336python/car-statistics/blob/master/car-statistics/app/models/files.py#L54 here for datetime generation used datetime module. And it should be in utils to provide it